app/controllers/users_controller.rb

```diff
@@ -0,0 +1,44 @@
+class UsersController < ApplicationController
+
+  def index
+    @users = User.all.order(:created_at)
+  end
+
+  def new
+    @user = User.new()
+  end
+
+  def show
+    @user = User.find_by(id: params[:id])
```

Comment:
You need to do something if the user doesn't exist.  Otherwise `@user.votes` is `nil`.

```suggestion
    @user = User.find_by(id: params[:id])
    if @user.nil?
      redirect_to root_path
      flash[:error] = "User not found"
      return
    end
```